### PR TITLE
gatsby-transformer-sharp: Added the option to use the original image

### PR DESCRIPTION
**Pull request by chiedo**
Users who don't want Sharp to process anything now have that option.

### CODE REVIEW

File: packages/gatsby-transformer-sharp/src/extend-node-type.js [hunk 1]
```diff
@@ -16,6 +16,11 @@ const {
 } = require(`gatsby-plugin-sharp`)
 
 const sharp = require(`sharp`)
+const {promisify} = require('util');
```
Comment (KyleAMathews) [inline packages/gatsby-transformer-sharp/src/extend-node-type.js]: This is a Node 8 only function. You can use https://www.npmjs.com/package/util.promisify

Comment (chiedo) [inline packages/gatsby-transformer-sharp/src/extend-node-type.js]: Just added!

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 188426f57c9cba13ce2d0a4268b41f40386e8950

https://app.netlify.com/sites/image-processing/deploys/596fd4bd8ebdd96cb87ffd70

**Comment by chiedo** [comment]
Done with my changes. Passing again in my local tests :)

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 188426f57c9cba13ce2d0a4268b41f40386e8950

https://app.netlify.com/sites/using-remark/deploys/596fd4be8ebdd96cb87ffd7c

**Comment by KyleAMathews** [comment]
Thanks!

**Pull request merged**